src/context.rs

```diff
@@ -80,6 +82,8 @@ struct ConcreteBaseAudioContextInner {
     queued_audio_listener_msgs: Mutex<Vec<ControlMessage>>,
     /// AudioListener fields
     listener_params: Option<AudioListenerParams>,
+    /// Denotes if this AudioContext is offline or not
+    offline: bool,
```

Comment:
We need to know this since the AudioDestinationNode has different behaviour for online and offline when changing channel counts